### ISSUE TITLE
关于网络架构询问

### ISSUE THREAD

**Issue by xuxiaoxxxx** [PaddlePaddle/PaddleSeg#1046]
请问paddleseg是否支持取出最后一步网络输出，就是在进行softmax前的一层，假如可以要如何取出结果？

**Comment by lingorX** [comment]
Models implemented in `paddleseg/models` will always return logits without softmax.
